### DISCUSSION TITLE
`appearance-none` on `select` multiple

### DISCUSSION THREAD

**Discussion by SHxKM** [tailwindlabs/tailwindcss#12844] [Help] [answered]
Seems like I'm missing something but this doesn't remove the right-side border from `select` elements. Example:

```
<select id="brand__in" name="brand__in" class="appearance-none border-none bg-none" multiple>
  <option value="Toyota">Toyota</option>
  <option value="Volvo">Volvo</option>
</select>
```

<img width="99" alt="SCR-20240127-cuqp" src="https://github.com/tailwindlabs/tailwindcss/assets/6573474/7b742fd6-75a7-4ceb-8985-9a8f337f6b24">

Any way to reset those styles?

**Comment by wongjn** [comment, marked as answer]
Can't quite tell from the image but could you be referring to the scrollbar? If so, you could:
- Apply an `overflow` value other than `scroll`.
- Hide the scrollbar itself via `::-webkit-scrollbar` pseudo element (for Chromium and Safari) and `scrollbar-width: none` for Firefox.

See these implementations in [this Tailwind Play](https://play.tailwindcss.com/D7XcWAPyqK).

**Comment by SHxKM** [comment]
> Can't quite tell from the image but could you be referring to the scrollbar?

Yes, and (obviously) your solution works perfectly. Thanks!
